samples/Mimban/Mimban.Server/Program.cs

```diff
@@ -124,6 +124,10 @@ await using (var scope = app.Services.CreateAsyncScope())
 {
     var manager = scope.ServiceProvider.GetRequiredService<IOpenIddictApplicationManager>();
 
+    using var context = scope.ServiceProvider.GetRequiredService<DbContext>();
+    await context.Database.EnsureDeletedAsync();
```

Comment:
Not sure what you're referring to exactly. `EnsureDeleted`?

Reply:
Yes, `EnsureDeletedAsync()`.

Reply:
Ahhhhh yeah, my bad, we opted for a persistent SQLite DB rather than an in-memory one.

To fix that, you'll need to add "does it exist already?" checks as in the other examples:

https://github.com/openiddict/openiddict-samples/blob/b9c9601c757745976504a48efd4f3977dae542ff/samples/Velusia/Velusia.Server/Worker.cs#L28-L60